src/integration-test/java/com/scalar/db/storage/dynamo/DynamoIntegrationTest.java

```diff
@@ -154,6 +156,50 @@ public void get_GetWithProjectionsGiven_ShouldRetrieveSpecifiedValues()
     assertThat(actual.get().getValue(COL_NAME5).isPresent()).isFalse();
   }
 
+  @Test
+  public void get_GetGivenForIndexedColumn_ShouldGet() throws ExecutionException {
```

Comment:
This is just a comment. We have the same integration code for each storage implementation. I'm wondering if we can unify the same integration test code for all storage implementations.

Reply:
@brfrn169 Let's do it! I'll create a story for that.